Automattic/jetpack
# Search: record meter. Add basic file structures & dummy data

This PR adds the basic files and dummy data needed to display a simple record meter chart. This PR does not add the other record meter features such as the record count, notice box, or tests.


#### Changes proposed in this Pull Request:

Adds the following files:
* bar-chart.jsx - the barchart component
* lib/create-data.jsx - a temporary file to generate dummy data
* lib/record-info.jsx - a lib file to convert data into the correct format to use with chartJS
* test/index.test.jsx
* test/record-info.test.jsx

#### Jetpack product discussion
Project board: pcNPJE-na

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

Visit `/wp-admin/admin.php?page=jetpack-search&features=record-meter` and make sure a chart is displaying at the bottom of the page. 

Check unit tests all pass

![image](https://user-images.githubusercontent.com/30754158/156617286-023fff68-15a6-4486-9272-5aa02cc03364.png)
